### PR TITLE
Add data directory as volume

### PR DESCRIPTION
**Pull request by mariusgrigaitis**
According to best practices:

https://docs.docker.com/v1.8/articles/dockerfile_best-practices/#volume

### CODE REVIEW

**Comment by lphoward** [comment]
Marius,

The traditional use case for fake-s3 has been unit testing, and related forms of testing, where storage ephemerality is expected.  Other use cases might value persistence, but Docker supports those situations without the need to make this choice for everyone.

For example. if you want a container to use a volume, then

    $ docker run --name my_s3 -v /fakes3_root -d lphoward/fake-s3

The fake-S3 root directory will then be added as a volume on the Docker host.

    $ docker inspect --format "{{range .Mounts}}{{.Source}}{{end}}" my_s3

In summary, a "best practice" is rarely best in every circumstance.  For fake-s3, I expect the proposal would be disruptive and undesirable to "the many," while others can easily fill the need as shown above.

Always open to what I may have missed here, so jump in.

Regards
